device/nokia/x86_64-nokia_ixr7250e_sup-r0/installer.conf

```diff
@@ -1,4 +1,4 @@
 CONSOLE_PORT=0x2f8
 CONSOLE_DEV=0
 CONSOLE_SPEED=115200
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="amd_iommu=off pci=resource_alignment=26@00:01.4 pci=noaer"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="amd_iommu=off pci=resource_alignment=26@00:01.4 pci=noaer crashkernel=8G-:1G"
```

Comment:
@jon-nokia  are you planning to remove the **noaer** in a different PR ?

Reply:
yes.  that will go into a different PR that may need corresponding NDK changes.